scripts/express/app.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-console */
+const express = require('express');
+const path = require('path');
+const shell = require('shelljs');
+
+const app = express();
+const port = process.env.PORT || 8081;
+// npm script to build assets by webpack
+const buildCommand = 'npm run compile:prod';
```

Comment:
For now, I'm fine with dev environment with production webpack. As webpack will eat a lot of memories, more dev features will make reviews app more unstable. I'd like to add more features if we can reduce the memory cost with webpack compile.